### PR TITLE
minia 3.2.0

### PR DESCRIPTION
**Pull request by tseemann**
- [ ] Have you followed the [guidelines for contributing](https://github.com/brewsci/homebrew-bio/blob/master/CONTRIBUTING.md)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/brewsci/homebrew-bio/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source FORMULA`, where `FORMULA` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict FORMULA` (after doing `brew install FORMULA`)?

-----

### CODE REVIEW

**Comment by tseemann** [comment]
Any ideas @rchikhi ?

```
[ 74%] Building CXX object ext/gatb-core/src/CMakeFiles/gatbcore-static.dir/gatb/template/TemplateSpecialization2.cpp.in_96.cpp.o
cd /tmp/minia-20181020-4158-piuwql/minia-v3.2.0-Source/build/ext/gatb-core/src && /home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/shims/linux/super/g++-5  -DGIT_SHA1="\"fatal: Not a git repository (or any of the parent directories): .git\"" -I/tmp/minia-20181020-4158-piuwql/minia-v3.2.0-Source/build/ext/gatb-core/include -I/tmp/minia-20181020-4158-piuwql/minia-v3.2.0-Source/build/ext/gatb-core/include/Release -I/tmp/minia-20181020-4158-piuwql/minia-v3.2.0-Source/thirdparty/gatb-core/gatb-core/src -I/tmp/minia-20181020-4158-piuwql/minia-v3.2.0-Source/thirdparty/gatb-core/gatb-core/thirdparty  -msse2 -msse4.2 -mpopcnt -DNDEBUG   -std=c++11 -O3 -DNDEBUG -Wall -Wno-unused-function -Wno-format -Wno-unknown-pragmas -Wno-invalid-offsetof -o CMakeFiles/gatbcore-static.dir/gatb/template/TemplateSpecialization2.cpp.in_96.cpp.o -c /tmp/minia-20181020-4158-piuwql/minia-v3.2.0-Source/build/ext/gatb-core/src/gatb/template/TemplateSpecialization2.cpp.in_96.cpp
{standard input}: Assembler messages:
{standard input}:52640: Error: unknown pseudo-op: `.uleb1'
{standard input}:52494: Error: invalid operands (*UND* and .gcc_except_table sections) for `-'
g++-5: internal compiler error: Killed (program cc1plus)
Please submit a full bug report,
with preprocessed source if appropriate.
See <file:///usr/share/doc/gcc-5/README.Bugs> for instructions.
ext/gatb-core/src/CMakeFiles/gatbcore-static.dir/build.make:117: recipe for target 'ext/gatb-core/src/CMakeFiles/gatbcore-static.dir/gatb/template/TemplateSpecialization10.cpp.in_128.cpp.o' failed
make[2]: *** [ext/gatb-core/src/CMakeFiles/gatbcore-static.dir/gatb/template/TemplateSpecialization10.cpp.in_128.cpp.o] Error 4
make[2]: *** Waiting for unfinished jobs....
make[2]: Leaving directory '/tmp/minia-20181020-4158-piuwql/minia-v3.2.0-Source/build'
CMakeFiles/Makefile2:194: recipe for target 'ext/gatb-core/src/CMakeFiles/gatbcore-static.dir/all' failed
make[1]: *** [ext/gatb-core/src/CMakeFiles/gatbcore-static.dir/all] Error 2
make[1]: Leaving directory '/tmp/minia-20181020-4158-piuwql/minia-v3.2.0-Source/build'
Makefile:154: recipe for target 'all' failed
make: *** [all] Error 2
```

File: Formula/minia.rb [hunk 1]
```diff
@@ -14,23 +14,27 @@ class Minia < Formula
   depends_on "cmake" => :build
   depends_on "zlib" unless OS.mac?
 
+  needs :cxx11
+
   def install
+    # Reduce memory usage for CircleCI.
+    ENV["MAKEFLAGS"] = "-j8" if ENV["CIRCLECI"]
```
Comment (sjackman) [inline Formula/minia.rb]: You can remove these two lines. The default is now four threads.

**Comment by sjackman** [comment]
Thanks, Torsten!

File: Formula/minia.rb (hunk 1 above)
Comment (tseemann) [reply Formula/minia.rb]: Will we do a bulk removal from all recipes?

**Comment by tseemann** [comment]
@sjackman i don't know what to do about the linux build failure. i will push again in case it was a build problem that day.

**Comment by tseemann** [comment]
@sjackman i don't know what to do about the linux build failure. i will push again in case it was a build problem that day.

**Comment by sjackman** [comment]
> Why is /usr/bin/gcc: 5.4.0 @sjackman ?

`/usr/bin/gcc` is 5.4.0 on Ubuntu 16.04 LTS (Xenial Xerus), which Brewsci uses for bottling.

**Comment by sjackman** [comment]
> Why is /usr/bin/gcc: 5.4.0 @sjackman ?

`/usr/bin/gcc` is 5.4.0 on Ubuntu 16.04 LTS (Xenial Xerus), which Brewsci uses for bottling.

**Comment by sjackman** [comment]
> Why is /usr/bin/gcc: 5.4.0 @sjackman ?

`/usr/bin/gcc` is 5.4.0 on Ubuntu 16.04 LTS (Xenial Xerus), which Brewsci uses for bottling.

**Comment by sjackman** [comment]
> Why is /usr/bin/gcc: 5.4.0 @sjackman ?

`/usr/bin/gcc` is 5.4.0 on Ubuntu 16.04 LTS (Xenial Xerus), which Brewsci uses for bottling.

**Comment by sjackman** [comment]
> Why is /usr/bin/gcc: 5.4.0 @sjackman ?

`/usr/bin/gcc` is 5.4.0 on Ubuntu 16.04 LTS (Xenial Xerus), which Brewsci uses for bottling.

**Comment by sjackman** [comment]
> Why is /usr/bin/gcc: 5.4.0 @sjackman ?

`/usr/bin/gcc` is 5.4.0 on Ubuntu 16.04 LTS (Xenial Xerus), which Brewsci uses for bottling.

**Comment by sjackman** [comment]
> Why is /usr/bin/gcc: 5.4.0 @sjackman ?

`/usr/bin/gcc` is 5.4.0 on Ubuntu 16.04 LTS (Xenial Xerus), which Brewsci uses for bottling.

File: Formula/minia.rb (hunk 1 above)
Comment (sjackman) [reply Formula/minia.rb]:
```
g++-5: internal compiler error: Killed (program cc1plus)
```
This error usually indicates that the compiler exceed available memory. Try reducing `-j8` to `-j4` (which is the default) or `-j2` if that doesn't work.

Comment (sjackman) [reply Formula/minia.rb]:
```
g++-5: internal compiler error: Killed (program cc1plus)
```
This error usually indicates that the compiler exceed available memory. Try reducing `-j8` to `-j4` (which is the default) or `-j2` if that doesn't work.

Comment (sjackman) [reply Formula/minia.rb]:
```
g++-5: internal compiler error: Killed (program cc1plus)
```
This error usually indicates that the compiler exceed available memory. Try reducing `-j8` to `-j4` (which is the default) or `-j2` if that doesn't work.

Comment (sjackman) [reply Formula/minia.rb]:
```
g++-5: internal compiler error: Killed (program cc1plus)
```
This error usually indicates that the compiler exceed available memory. Try reducing `-j8` to `-j4` (which is the default) or `-j2` if that doesn't work.

**Comment by rchikhi** [comment]
Indeed minia's building process is quite memory intensive due to our heavy use of C++ template specialization (optimizations for various kmer sizes).

**Comment by rchikhi** [comment]
Indeed minia's building process is quite memory intensive due to our heavy use of C++ template specialization (optimizations for various kmer sizes). Thanks for putting minia on homebrew @tseemann !

**Comment by rchikhi** [comment]
Indeed minia's building process is quite memory intensive due to our heavy use of C++ template specialization (optimizations for various kmer sizes). Thanks for putting minia on homebrew @tseemann !

**Comment by rchikhi** [comment]
Indeed minia's building process is quite memory intensive due to our heavy use of C++ template specialization (optimizations for various kmer sizes). Thanks for putting minia on homebrew @tseemann !

**Comment by rchikhi** [comment]
Indeed minia's building process is quite memory intensive due to our heavy use of C++ template specialization (optimizations for various kmer sizes). Thanks for putting minia on homebrew @tseemann !

**Comment by rchikhi** [comment]
Indeed minia's building process is quite memory intensive due to our heavy use of C++ template specialization (optimizations for various kmer sizes). Thanks for putting minia on homebrew @tseemann !

**Comment by rchikhi** [comment]
Indeed minia's building process is quite memory intensive due to our heavy use of C++ template specialization (optimizations for various kmer sizes). Thanks for putting minia on homebrew @tseemann !

**Comment by rchikhi** [comment]
Indeed minia's building process is quite memory intensive due to our heavy use of C++ template specialization (optimizations for various kmer sizes). Thanks for putting minia on homebrew @tseemann !

**Comment by sjackman** [comment]
> Will we do a bulk removal from all recipes?

Could do, but I'm more inclined to do it by attrition, and remove it as formulae are updated with new versions.

**Comment by tseemann** [comment]
Reducing CIRCLE threads / RAM worked - thanks for the tip @sjackman

**Comment by tseemann** [comment]
Arggh
```
==> Fetching patch (bottle commit)
Patch: https://github.com/LinuxbrewTestBot/homebrew-bio/compare/brewsci:master...pr-499.patch

curl: (22) The requested URL returned error: 404 Not Found
Error: Failure while executing; `/usr/bin/curl -q --show-error --user-agent Linuxbrew/1.7.7-123-g8f5de98\ \(Linux\;\ x86_64\ \"Red\ Hat\ Enterprise\ Linux\ Server\ release\ 7.5\ \(Maipo\)\"\)\ curl/7.29.0 --fail --progress-bar --location --remote-time --continue-at 0 --output /home/linuxbrew/.cache/Homebrew/brewsci:master...pr-499.patch https://github.com/LinuxbrewTestBot/homebrew-bio/compare/brewsci:master...pr-499.patch` exited with 22. Here's the output:

curl: (22) The requested URL returned error: 404 Not Found
```

**Comment by tseemann** [comment]
```
18860   /home/linuxbrew/.linuxbrew/Cellar/minia/3.2.0/bin
4       /home/linuxbrew/.linuxbrew/Cellar/minia/3.2.0/INSTALL_RECEIPT.json
4       /home/linuxbrew/.linuxbrew/Cellar/minia/3.2.0/lib
36      /home/linuxbrew/.linuxbrew/Cellar/minia/3.2.0/LICENSE
4       /home/linuxbrew/.linuxbrew/Cellar/minia/3.2.0/README.md
295080  /home/linuxbrew/.linuxbrew/Cellar/minia/3.2.0/te
```

i'll remove the test folder?

**Comment by sjackman** [comment]
> i'll remove the test folder?

Sounds good.

**Pull request closed without merging**